components/clp-package-utils/clp_package_utils/scripts/del_archives.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import logging
+import pathlib
```

Comment:
* Let's use `from pathlib import Path` and then replace the instances of `Path` below.
* Ditto for the other `del_archives.py`.